### DISCUSSION TITLE
How to create a new object lifecycle management rule?

### DISCUSSION THREAD

**Discussion by qiuyuyin** [seaweedfs/seaweedfs#4108] [Q&A] [answered]
**we want to create a new object lifecycle management rule that transition objects from a MinIO bucket to a remote storage tier on the Amazon Web Services S3 storage backend or an S3-compatible service.**
Here is the minio' s realization:
https://min.io/docs/minio/linux/administration/object-management/transition-objects-to-s3.html

**Comment by chrislusf** [comment, marked as answer]
you can do the same with https://github.com/seaweedfs/seaweedfs/wiki/Gateway-to-Remote-Object-Storage

**Reply by qiuyuyin** [reply]
can we imply the function of transition, instead of setting the mirror from the local?

**Reply by chrislusf** [reply]
I do not get what you mean.

**Reply by qiuyuyin** [reply]
Sorry, I already know what to do. I get it.

**Reply by qiuyuyin** [reply]
thanks for your time
